scripts/build.sh

```diff
@@ -32,6 +33,10 @@ while [[ $# -gt 0 ]]; do
       configuration=$2
       shift 2
       ;;
+    --dotnetInstallDirectory)
+      dotnetInstallDirectory=$2
+      shift 2
+      ;;
```

Comment:
What's this doing?

Reply:
It specifies the dotnet install directory which is prepended to the PATH environment variable. Considering in CI this is done by the docker image, I could probably remove this safely.